ceno_zkvm/src/instructions/riscv/slt.rs

```diff
@@ -244,10 +246,9 @@ mod test {
 
     #[test]
     fn test_sltu_random() {
-        let mut rng = rand::thread_rng();
-        let a: u32 = rng.gen();
-        let b: u32 = rng.gen();
-        verify::<SltuOp>("random 1", a, b, (a < b) as u32);
-        verify::<SltuOp>("random 2", b, a, (a >= b) as u32);
+        proptest::proptest!(|(a in any::<u32>(), b in any::<u32>())| {
```

Comment:
Thanks for taking this on!

Please see https://github.com/scroll-tech/ceno/pull/667 for something similar.

The main difference is `any::<u32>` picks from all `u32` uniformly at random, but that PR is slightly more clever about the sampling.

Reply:
Oh I missed looking at that, sorry

Reply:
No worries. I didn't advertise enough, I guess.